uint8 Roller
uint8 Mode
uint8 KeepOrNot
uint8 PlaceOrNot
string RobotStatus
